scarb/src/core/package/mod.rs

```diff
@@ -70,6 +70,10 @@ impl Package {
         self.manifest.targets.iter().any(Target::is_cairo_plugin)
     }
 
+    pub fn is_builtin(&self) -> bool {
+        self.manifest.targets.iter().any(Target::is_builtin)
+    }
```

Comment:
I think it's in line with the current approach and is quite similar to
```
    pub fn is_lib(&self) -> bool {
        self.manifest.targets.iter().any(Target::is_lib)
    }

    pub fn is_cairo_plugin(&self) -> bool {
        self.manifest.targets.iter().any(Target::is_cairo_plugin)
    }
```
above. 

It's also quite useful as it allows us to avoid lengthy: 
```
pkg.manifest.targets.iter().any(|x| x.is_builtin())
```

Reply:
> is_lib, is_cairo_plugin

Those are target kinds, not target props. 

> pkg.manifest.targets.iter().any(|x| x.is_builtin())

I don't see a problem with this. Separation of concerns is better this way. Target props do not define package objective,  target kinds do.

Reply:
Addressed in [af8dc5b](https://github.com/software-mansion/scarb/pull/1605/commits/af8dc5b82b2a768f1be5967858841d9c7ca28875)